This repository contains analyses of proteomics data for Liver Hepatocellular carcinoma (LIHC).

- Proteomics data for LIHC was downloaded from NCI proteomics data commons (PDC), PDC Study ID: PDC000198, study by Gao et al 2019 (https://doi.org/10.1016/j.cell.2019.08.052).
- Analysis was aimed at recreating Figure 3.
- Figure 3A: Heatmap of consensus clustering of 1,274 proteins differentially expressed between tumor and adjacent non-tumor tissue  (SAM, FDR q < 0.05, fold change > 1.5). Rows = proteins; columns = tumors. Z-score scaled log2 relative abundances.  Samples grouped into three proteomic subgroups: S-Mb (metabolic),  S-Pf (proliferative), and S-Me (microenvironment dysregulated).
- Figure 3B: Kaplan–Meier overall survival curves comparing the three proteomic subgroups. Note: The notes in the excel file downloaded from NCI-PDC mentions that clinical data for 9 patients were lost and not available.
- Figure 3E & F: Heatmap of significantly differentially expressed proteins in tumors with or without thrombus (3E) and barplot of significantly enriched biological processes (with down regulated genes) in tumors with thrombus. The authors have 82 genes on the heatmap (FDR cut off used by authors is not clear as they mention different numbers in the figure legend and methods ) while I got 281 (FDR cutoff of 0.1). This difference can also be attributed to imputation methods used for imputing missing values (The authors used an older pamr version, I used knn.impute). The barplot for enriched GO terms (biological processes) encompasses pathways captured by the authors. Full list of enriched GO terms pathways is given presented in GO_BP_results.csv file.
